src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -846,7 +859,8 @@ internal Binder VisitCompilationUnit(CompilationUnitSyntax compilationUnit, bool
                 Binder result;
                 if (!binderCache.TryGetValue(key, out result))
                 {
-                    result = this.buckStopsHereBinder;
+                    var buckStopsHereBinder = this.buckStopsHereBinder;
+                    result = buckStopsHereBinder;
```

Comment:
Is the local `buckStopsHereBinder` needed?

Reply:
> Is the local buckStopsHereBinder needed? (Can this change be reverted?)

Yes, will remove the local.

---
In reply to: [384695860](https://github.com/dotnet/roslyn/pull/41926#discussion_r384695860) [](ancestors = 384695860)